tests/functional/validate/76-host-to-platform-upgrade-dummy-mode.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
This shouldn't be in the validate tests, move in with the run tests or put somewhere else.

Reply:
(maybe a new "platforms" test sub-dir?)

Reply:
how about a `upgrade7to8` folder - if used correctly it'll help us know whether we've deprecated the rather involved upgrade syntax?